delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/StoragesConfig.scala

```diff
@@ -63,12 +63,16 @@ object StoragesConfig {
       amazon: Option[S3StorageConfig],
       remoteDisk: Option[RemoteDiskStorageConfig]
   ) {
-    def get(tpe: StorageType): Option[StorageTypeEntryConfig] =
-      tpe match {
-        case StorageType.DiskStorage       => Some(disk)
-        case StorageType.S3Storage         => amazon
-        case StorageType.RemoteDiskStorage => remoteDisk
-      }
+    def showFileLocation: ShowFileLocation =
+      ShowFileLocation(disk.showLocation, remoteDisk.exists(_.showLocation), amazon.exists(_.showLocation))
+  }
+
+  final case class ShowFileLocation(disk: Boolean, remote: Boolean, s3: Boolean) {
```

Comment:
maybe a set of storage type instead of these 3 booleans ?
And then, you call contains ?

Reply:
Ok, done